cw-orch-daemon/src/state.rs

```diff
@@ -167,6 +167,25 @@ impl DaemonState {
         serde_json::to_writer_pretty(File::create(&self.json_file_path).unwrap(), &json)?;
         Ok(())
     }
+
+    /// Flushes all the state related to the current chain
+    /// Only works on Local networks
+    pub fn flush(&self) -> Result<(), DaemonError> {
+        if self.chain_data.kind != ChainKind::Local {
+            panic!("Can only flush local chain state");
+        }
+        if self.read_only {
+            return Err(DaemonError::StateReadOnly);
+        }
+
+        let mut json = self.read_state()?;
+
+        json[&self.chain_data.network_info.chain_name][&self.chain_data.chain_id.to_string()] =
```

Comment:
I can see it be running at the start of "local" scripts. I think this will panic if there are no local state yet. I think we should try to do multiple `.get_mut()` and do nothing if there are no state

Reply:
This doesn't panic (it just returns Value::Null) :)

Reply:
But let me make sure

Reply:
This doesn't panic, added to the examples if you want to run it

Reply:
> This doesn't panic (it just returns Value::Null) :)

Oh, that's nice, didn't know that, sorry